superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/ag-grid/AGGridViz.tsx

```diff
@@ -372,10 +374,10 @@ export default function AGGridViz({
               ...(selectedData.typeData.DATETIME || []),
             ],
           }}
-          disabled={selectedData.typeData.harmonized_email_id.length > 30}
+          disabled={selectedData.typeData.harmonized_email_id.length > RETENTION_LIMIT}
           tooltip={
-            selectedData.typeData.harmonized_email_id.length > 100
-              ? 'Cannot retain more than 100 EML IDs.'
+            selectedData.typeData.harmonized_email_id.length > RETENTION_LIMIT
+              ? 'Cannot retain more than RETENTION_LIMIT harmonized email IDs.'
```

Comment:
This is just me not knowing JS. Does this render properly? Nothing needed to make RETENTION_LIMIT not be interpreted as a string?

Reply:
You are correct. Something like this should work:
```suggestion
              ? `Cannot retain more than ${RETENTION_LIMIT} harmonized email IDs.`
```